plugins/org.locationtech.udig.tool.measure/src/org/locationtech/udig/tool/measure/internal/messages_de.properties

```diff
@@ -1,2 +1,2 @@
-DistanceTool_distance = Entfernung \= {0}
+DistanceTool_distance = Entfernung \= {0} (letztes segment \= {1})
```

Comment:
What do you think about "Gesamtstrecke \= {0} (aktuelles Segment \= {1})"

Reply:
OK modification commited after all you are the German guy :)